packages/devtools_test/lib/src/wrappers.dart

```diff
@@ -49,17 +49,6 @@ Widget wrap(Widget widget) {
   );
 }
 
-Widget wrapWithAnalytics(
```

Comment:
add this back when you add the analytics prompt test back

Reply:
I moved it to the test, because it is not used in production code.